resolwe_bio/kb/elastic_indexes.py

```diff
@@ -54,3 +81,35 @@ def process_object(self, obj, push=True):
 
         """
         return super(FeatureSearchIndex, self).process_object(obj, push=False)
+
+    def get_autocomplete_value(self, obj):
+        """Return autocomplete value."""
+        return [
+            obj.feature_id,
+            obj.name
+        ] + obj.aliases
+
+
+class MappingSearchDocument(BaseDocument):
+    """Index for mapping search."""
+
+    # pylint: disable=no-member
+    relation_type = dsl.String(index='not_analyzed')
+    source_db = dsl.String(index='not_analyzed')
+    source_id = dsl.String(index='not_analyzed')
+    target_db = dsl.String(index='not_analyzed')
+    target_id = dsl.String(index='not_analyzed')
+    relation_type = dsl.String(index='not_analyzed')
+
+    class Meta:
+        """Meta class for mapping search document."""
+
+        index = 'mapping_search'
+
+
+class MappingSearchIndex(BaseIndex):
```

Comment:
Use bulk push:
* override `process_object` with `push=False`
* manually push documents in management command and tests
(see `FeatureSearchIndex` and `insert_features` command for example)